### PR TITLE
Fixed x overflow in mobile view

### LINKED ISSUE #135: Mobile view x overflow
When we scroll horizontally in the mobile view, we can see the overflow of the page:

https://user-images.githubusercontent.com/50210213/134698540-26305757-754d-4e63-82cb-34032f21f10d.mp4

We need to remove the x overflow.

### PR DESCRIPTION
**Pull request by TiagooGomess**
Closes #135

### CODE REVIEW

**Comment by TiagooGomess** [comment]
I don't know if this is the better approach to fix this problem, I need some feedback.

**Comment by imnotteixeira** [comment]
This is a hack, please find the element that is triggering the overflow and fix that instead

**Comment by imnotteixeira** [comment]
> This is a hack, please find the element that is triggering the overflow and fix that instead

From what I could gather, it's the Grid from ProductDescription

**Comment by imnotteixeira** [comment]
The fix is to remove the `spacing={10}` from the outer Grid container, but that will make the items be closer together, so you need to add padding to them specifically, to maintain the visual aspect

File: src/components/HomePage/ProductPlacementArea/productDescriptionStyles.js [hunk 1]
```diff
@@ -23,5 +23,9 @@ export default makeStyles(() => ({
     companyName: {
         fontWeight: "bold",
     },
-}))
-;
+    productDescriptionGrid: {
+        "& > div": {
+            padding: isMobile ? "40px 0" : "40px 40px",
```
Comment (imnotteixeira) [inline src/components/HomePage/ProductPlacementArea/productDescriptionStyles.js:28]: Use theme.spacing instead of hardcoded 40px

**Review by imnotteixeira — CHANGES_REQUESTED** [review]

**Review by imnotteixeira — APPROVED** [review]

**Pull request merged**